### ISSUE TITLE
Blur and other settings should be treaten globally for the desktop client

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/talk-desktop#156]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

When using things like backgorund blur, which camera, etc. we store some of the information depending on the conversation token. For some things like Camera/Mic on/off that makes sense to keep on a per-conversation-level.
But e.g. whether or not to blur your background could/should be globally. I think we just followed the existing pattern when introducing it and since it depends on your hardware whether it's okay/good to do that, we could not store it in the talk settings. But for the desktop client the hardware barely changes, so I think it would be good to have it as a global setting.

* Might depend on #27

**Comment by ShGKme** [comment]
@nickvergessen do you think it is still actual?

**Comment by nickvergessen** [comment]
Yes, it's the most annoying setting we have. It's bothering me everytime there is a new room and I join the call for the first time, that I need to visit the settings and re-blur my background. But I think this should be solvable on web and the desktop will benefit from it automatically.
